common/level.go

```diff
@@ -27,26 +24,26 @@ func (t *tile) Width() float32 {
 }
 
 func (t *tile) Texture() *gl.Texture {
-	return t.Image.Texture()
+	return t.Image.id
 }
 
 func (t *tile) Close() {
 	// noop
 }
 
 func (t *tile) View() (float32, float32, float32, float32) {
-	return t.Image.u, t.Image.v, t.Image.u2, t.Image.v2
+	return t.Image.View()
 }
 
 type tile struct {
-	Point
+	engo.Point
 	height int
 	width  int
-	Image  *Region
+	Image  *Texture
 }
 
 type tilesheet struct {
-	Image    *Texture
+	Image    *TextureResource
```

Comment:
You can also use `Texture` instead; since you're not using anything `TextureResource` specific (and `TextureResource` is a wrapper for `Texture`). 

It looks like we're not using `tilesheet` anyway? Nor are we using `createTileset`? (nor could anyone else because they're unexported? Correct me if I'm wrong here.